src/spec-node/collectionCommonUtils/packageCommandImpl.ts

```diff
@@ -171,8 +171,9 @@ export async function packageCollection(args: PackageCommandInput, collectionTyp
 			const devcontainerJsonName = `devcontainer-${collectionType}.json`;
 			const jsonPath = path.join(tmpSrcDir, devcontainerJsonName);
 			if (!(await isLocalFile(jsonPath))) {
-				output.write(`${collectionType} '${c}' is missing a ${devcontainerJsonName}`, LogLevel.Error);
-				return;
+				output.write(`(!) WARNING: ${collectionType} '${c}' is missing a ${devcontainerJsonName}. Skipping... `, LogLevel.Error);
+				await rmLocal(tmpSrcDir, { recursive: true, force: true });
```

Comment:
Would it make sense to check for the file before copying the folder? You could avoid copying and removing it.

Reply:
Good point! Modified